Authentication/AuthenticationBootstrapper.swift

```diff
@@ -219,33 +235,29 @@ public class AuthenticationBootstrapper<User: UserType, SessionService: SessionS
 extension AuthenticationBootstrapper: LoginControllerTransitionDelegate {
     
     public func onSignup(controller: LoginController) {
```

Comment:
I like `onSignupButtonPressed`, but as is the transition delegate it may be good to state in the name of the function that they must do the transition. So maybe `toSignup` or `performTransitionToSignup` are better.